ddtrace/_trace/context.py

```diff
@@ -0,0 +1,268 @@
+import base64
```

Comment:
Have you considered combining the `_trace` and `tracing` directories? What's the distinction between them meant to be?

(It's a decision that was made elsewhere, but it's the first time I'm seeing it.)

Reply:
I put it on my todo list, thanks!